ui/sign-in.reel/sign-in.html

```diff
@@ -71,13 +71,25 @@
                 "element": {"#": "submit"},
                 "value": "Sign In"
             }
+        },
+
+        "browserMessage": {
+            "prototype": "montage/ui/text.reel",
+            "properties": {
+                "element": {"#": "browserMessage"},
+                "value": "We're sorry but at this time FreeNAS only supports the Chrome browser."
+            },
+            "bindings": {
+                "classList.has('hide')": {"<-": "@owner.isBrowserSupported"}
```

Comment:
I have no strong opinion, but I think we mainly used is-* or has-* class names so far, which kept them consistent with properties names.
We may consider `hide` as an exception, as it is a class offered directly by montage, though...